ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.HaukkeManor,
+  triggers: [
+    {
+      id: 'Haukke Normal Dark Mist DPS/Heals',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'], capture: false }),
+      condition: function(data) {
+        return data.role != 'tank';
+      },
+      suppressSeconds: 2,
+      response: Responses.getOut('alert'),
+    },
+    {
+      id: 'Haukke Normal Dark Mist Tank',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'] }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      suppressSeconds: 2,
+      response: Responses.stun('info'),
+    },
+    {
+      id: 'Haukke Normal Claviger Void Fire II DPS/Heals',
+      netRegex: NetRegexes.startsUsing({ id: '357', source: 'Manor Claviger', capture: false }),
+      condition: function(data) {
+        return data.role != 'tank';
+      },
+      response: Responses.getOut('alert'),
+    },
+    {
+      id: 'Haukke Normal Claviger Void Fire II Tank',
+      netRegex: NetRegexes.startsUsing({ id: '357', source: 'Manor Claviger' }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      response: Responses.interrupt('info'),
+    },
+    {
+      id: 'Haukke Normal Steward Soul Drain DPS/Heals',
+      netRegex: NetRegexes.startsUsing({ id: '35C', source: 'Manor Steward', capture: false }),
+      condition: function(data) {
+        return data.role != 'tank';
+      },
+      response: Responses.outOfMelee('alert'),
+    },
+    {
+      id: 'Haukke Normal Steward Soul Drain Tank',
+      netRegex: NetRegexes.startsUsing({ id: '35C', source: 'Manor Steward' }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      id: 'Haukke Normal Amandine Void Fire III',
+      netRegex: NetRegexes.startsUsing({ id: '356', source: 'Lady Amandine' }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      response: Responses.interrupt('info'),
+    },
+    {
+      id: 'Haukke Normal Amandine Void Thunder III',
+      netRegex: NetRegexes.startsUsing({ id: '358', source: 'Manor Claviger' }),
+      condition: function(data) {
+        return data.role == 'tank';
+      },
+      response: Responses.interrupt('info'),
+    },
+    {
+      id: 'Haukke Normal Void Lamps',
+      regex: /The void lamps have begun/,
```

Comment:
There is actually a [`netRegex` helper](https://github.com/quisquous/cactbot/blob/0712f53cdde5ae5a8e49fe22cf82c2c8a9047d45/resources/netregexes.js#L286) for this too!
```suggestion
      netRegex: NetRegexes.message({ line: 'The void lamps have begun emitting an eerie glow*?' }),
```

Typically, unless there's a specific reason not to, it's recommended that you use the helpers instead of bare regexes. It makes reviewing and debugging easier.